docs/setup-instructions.md

```diff
@@ -0,0 +1,50 @@
+## Setup Instructions
+
+Please check our prerequisites [here](https://github.com/ARPA-Network/BLS-TSS-Network/blob/main/docs/eigenlayer-onboarding.md#prerequisites).
+
+### Step 1: Clone the repo
+
+```bash
+git clone https://github.com/ARPA-Network/ARPA-Network-AVS-Monitoring.git
+```
+
+### Step 2: Update configuration
+
+Update `config.yml` file in the root directory.
+
+### Step 3: Run setup script
+
+```bash
+chmod +x setup.sh
+./setup.sh
+```
+
```

Comment:
Add something here like: 

This will create the following two files which are used by the custom-exporter and cloudwatch-exporter containers:
- aws_exporter_config_example.yml
- custom-exporter/exporter-config.yml

You can always change your node config at a later time by doing the following
- Update config.yml with new node info.
- Re-run the setup script.
- Restart your docker container with docker compose.